app/Kconfig

```
@@ -477,6 +477,10 @@ if ZMK_USB_LOGGING
 config ZMK_LOG_LEVEL
 	default 4
 
+# We do this to avoid log loop where logging to USB generates more log messages.
+config USB_CDC_ACM_LOG_LEVEL
+    default 1
```

Consistency with the rest of the file
```suggestion
	default 1
```

---

Fixed.